src/struct/TwitchEmote.js

```diff
@@ -53,6 +53,35 @@ class TwitchEmote extends Emote {
     toLink(size = 0) {
         return Constants.Twitch.CDN(this.id, size); // eslint-disable-line new-cap
     }
+
+    /**
+     * Override for `toJSON`.
+     * Will result in a JSON representation of a TwitchEmote
+     * @returns {Object}
+     */
+    toJSON() {
+        return Object.assign({}, super.toJSON(), {
+            animated: this.animated,
+            set: this.set,
+            type: this.type
+        });
+    }
+
+    /**
+     * Converts a JSON into a TwitchEmote
+     * @param {JSON} [emoteJSON] - JSON representation of this emote
```

Comment:
It should probably be of type EmoteJSON no?

Reply:
No? I didn't see an EmoteJSON class or type of object in the code. Note that this is used in the docs and is not linked to the TS typings.
As I understand, we're passing to `fromJSON()` an already parsed JSON file, which is an Object or Array-like.

Also, since the JSON was already parsed... maybe the functions should be named `fromObject` and `toObject`? 
This would match what they require/return and the export could actually be used in other forms than JSON (like for storage in a DB).

Reply:
Oh yeah silly me! A bit new to type hints in js :P